src/lib.rs

```diff
@@ -1934,34 +1933,13 @@ impl<'a> Drop for SetLenOnDrop<'a> {
     }
 }
 
-#[cfg(feature = "const_generics")]
```

Comment:
I'd like to keep use of const generics behind an optional feature in smallvec 1.x, so it can continue compiling on Rust versions older than 1.51.  Let's wait until 2.0 to remove this feature completely.

Reply:
Ooopsss... My bad :p